src/components/picker/types.ts

```diff
@@ -159,6 +161,49 @@ type PickerExpandableOverlayProps = {
   enableModalBlur?: boolean;
 };
 
+interface PickerSelectionStatusLabelData {
+  selectedCount: number;
+  value: PickerMultiValue;
```

Comment:
Do we actually need to send this `value` to the user?

Reply:
when using Multi picker the onChange get's called only when the Done button is pressed.
 I think we should align with that so the value will be visible to the user via the `onPress` functionality.
 Removing this `value`.

Reply:
done, moved `selectedCount` into `usePickerSelection`